### DISCUSSION TITLE
Does pestpp-ies behave the same as pestpp-swp when noptmax=0?

### DISCUSSION THREAD

**Discussion by hjia1005** [usgs/pestpp#287] [Q&A] [answered]
Just a very simple question - does pestpp-ies behave the same as pestpp-swp when noptmax=0 if I provide the parameter ensembles / realizations? The reason I am asking is, the rmr file from pestpp-ies includes both realization_id and run_id for each completed model run, which is convenient when using panther_transfer_on_finish. However, rmr file from pestpp-swp only includes run_id, not sure how to relate the model outputs to their corresponding parameter sets.

**Comment by rhugman** [comment, marked as answer]
Pestpp-ies with 'noptmax = -1' should replicate swp's behaviour, if all you want is to sweep through an ensemble of model runs.

**Reply by hjia1005** [reply]
Thanks Rui!
